conans/test/functional/build_helpers/meson_test.py

```diff
@@ -30,4 +30,5 @@ def run(self, cmd):
 
         client.save({"conanfile.py": conanfile_vcvars})
         client.run('create . pkg/1.0@ -e PATH="MyCustomPath"')
-        self.assertIn("pkg/1.0: PATH ENV VAR: MyCustomPath;", client.out)
+        self.assertIn("pkg/1.0: PATH ENV VAR: ", client.out)
```

Comment:
How is this test different than before? Is this change necessary?

Reply:
Previously, the environment variable was inserted in the middle, and the vs environment variable must be inserted earlier, otherwise it will be destroyed by the other environment of the system.

The resulting effect is
```
pkg/1.0: PATH ENV VAR: [vs env];bla bla bla;MyCustomPath;
```
Instead of expected
```
pkg/1.0: PATH ENV VAR: MyCustomPath;[vs env];bla bla bla;
```